### PR TITLE
Release 0.21.9

### PR DESCRIPTION
**Pull request by Stypox**
# APK for testing
From GitHub Actions: [NewPipe_v0.21.9_RC2.zip](https://github.com/TeamNewPipe/NewPipe/files/6961141/NewPipe_v0.21.9_RC2.zip)

**Please report _only new_ bugs** in #6841

# App changelog 

### New
- Added comments disabled functionality #6483 
- Player seekbar thumbnail preview #6434 
- Add ability to mark a feed item as watched #6773 
- Show hearts in comments #6741 

### Improved
- Improved metadata layout, better tags accessibility #6463 
- Various improvements to the player (e.g. faster closing) #6566
- Apply service theme to UI components #6456 #6771 #6847

### Fixed
- Made the thumbnail in the minimized player visible again #6616 
- Fixed 'Unrecoverable player error occurred' when rotating device while playing a video #6503
- Fix duplicate items in queue causing endless buffering #6712 
- Delete pending missions with invalid storage fixing a crash when starting a download #6721 #6778 
- Prevent ReCAPTCHA webview from keeping YouTube loaded in background #6733 
- Disable feed click events while refresh overlay is shown #6550 
- Fix channel item span count for SubscriptionFragment #6814 

### Development
- Upgraded Gradle #6538 
- Updated some libraries #6540, #6702
- Specify emulator-build version in CI job fixing emulator build #6560 
- Do not immediately commit preference changes on import #6531 
- Fix API level 29 from android ci tests #6678, #6715  
- Use GestureDetectorCompat #6637 
- Add instrumented tests for LocalPlaylistManager.createPlaylist #5531 #6706 
- Bump compileSdkVersion to 30 #6654
- Fix some build warnings #6688 
- Added a Kotlin section in CONTRIBUTING.md #6659 
- Use Kotlin methods in LicenseFragment #6675
- Use WindowInsetsCompat's getInsets() method #6689
- Consolidate edittext alert dialogs into one common layout #6720 
- Fix deprecated fill_parent attributes #6782
- Check formatting of kotlin files in CI #6808 
- Update some AndroidX libraries #6719
- Use NotificationChannelCompat #6633

### CODE REVIEW

**Comment by mhmdanas** [comment]
Off-topic, but since GitHub now shows a PR's title when linking it, maybe we don't have to repeat the titles anymore (I guess this might not be the case for other clients though, so it's not like doing this will be totally a good thing)?

**Comment by opusforlife2** [comment]
@mhmdanas Its own mobile website doesn't show the titles yet. So I think we should wait at least that long before revisiting this.

**Comment by Stypox** [comment]
And neither does the releases page

**Comment by TacoTheDank** [comment]
I noticed that #6536 is included in this release, but doesn't seem to be mentioned in the above changelog. Perhaps some others were missed as well? :thinking: Not that it's important, exactly, but it'd be more complete for the blog post.

**Comment by opusforlife2** [comment]
@TacoTheDank That PR was in 0.21.7.

**Comment by TacoTheDank** [comment]
Oh I see, ok haha

File: app/src/main/res/values-b+zh+HANS+CN/strings.xml [hunk 1]
```diff
@@ -126,7 +127,8 @@
     <string name="search_no_results">没有结果</string>
     <string name="no_subscribers">无人订阅</string>
     <plurals name="subscribers">
-        <item quantity="other">%s 人订阅</item>
+        <item quantity="one">%s 位订阅者</item>
+        <item quantity="other"/>
```
Comment (TobiGr) [inline app/src/main/res/values-b+zh+HANS+CN/strings.xml:131]: Why are those added every once in a while...

Comment (TobiGr) [reply app/src/main/res/values-b+zh+HANS+CN/strings.xml]: Done

**Review by TobiGr — APPROVED** [review]
